fulcio-rekor-kind.yml is broken

All runs of fulcio-rekor-kind.yml  are failing in "Create sample image" step:

```
go: creating new go.mod: module example.com/demo
2026/07/04 05:28:50 Using base cgr.dev/chainguard/static:latest@sha256:77d8b8925dc27970ec2f48243f44c7a260d52c49cd778288e4ee97566e0cb75b for example.com/demo
2026/07/04 05:28:51 current folder is not a git repository. Git info will not be available
2026/07/04 05:28:51 Building example.com/demo for linux/amd64
2026/07/04 05:29:02 Publishing registry.local:5000/knative/demo:latest
Error: failed to publish images: error publishing ko://example.com/demo: writing sbom: Get "https://registry.local:5000/v2/": http: server gave HTTP response to HTTPS client
Error: Process completed with exit code 1.
```

My guess is that the new ko release (0.19) is breaking this

--- a/.github/workflows/fulcio-rekor-kind.yaml
+++ b/.github/workflows/fulcio-rekor-kind.yaml
@@ -77,6 +77,36 @@ jobs:
 
     - uses: ko-build/setup-ko@d006021bd0c28d1ce33a07e7943d48b079944c8d # v0.9
 
+    - name: Set up ko wrapper for insecure registry
+      run: |
+        mkdir -p ${{ github.workspace }}/bin
+        cat <<'EOF' > ${{ github.workspace }}/bin/ko
+        #!/usr/bin/env bash
+        MY_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+        for dir in ${PATH//:/ }; do
+          if [ "$dir" != "$MY_DIR" ] && [ -x "$dir/ko" ]; then
+            REAL_KO="$dir/ko"
+            break
+          fi
+        done
+        if [ -z "$REAL_KO" ]; then
+          echo "Real ko not found" >&2
+          exit 1
+        fi
+        cmd="$1"
+        shift
+        case "$cmd" in
+          build|publish|resolve|apply)
+            exec "$REAL_KO" "$cmd" --insecure-registry "$@"
+            ;;
+          *)
+            exec "$REAL_KO" "$cmd" "$@"
+            ;;
+        esac
+        EOF
+        chmod +x ${{ github.workspace }}/bin/ko
+        echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+
     - uses: sigstore/cosign-installer@d7543c93d881b35a8faa02e8e3605f69b7a1ce62 # v3.10.0
 
     - name: Setup Cluster

--- a/.github/workflows/test-action-tuf.yaml
+++ b/.github/workflows/test-action-tuf.yaml
@@ -77,7 +77,7 @@ jobs:
           fmt.Println("hello world")
         }
         EOF
-        demoimage=$(ko publish -B example.com/demo-action-with-tuf)
+        demoimage=$(ko publish --insecure-registry -B example.com/demo-action-with-tuf)
         echo "demoimage=$demoimage" >> $GITHUB_ENV
         echo Created image $demoimage
         popd

--- a/.github/workflows/test-release.yaml
+++ b/.github/workflows/test-release.yaml
@@ -112,7 +112,7 @@ jobs:
           fmt.Println("hello world")
         }
         EOF
-        demoimage=`ko publish -B example.com/demo-with-release`
+        demoimage=`ko publish --insecure-registry -B example.com/demo-with-release`
         echo "demoimage=$demoimage" >> $GITHUB_ENV
         echo Created image $demoimage
         popd

